vivarium/core/store.py

```diff
@@ -1006,6 +1006,29 @@ def emit_data(self):
             return self.value
         return None
 
+    def toggle_emits(self, emit=False, paths=None):
+        """
+        Turn on/off emits for all inner nodes of the list of paths.
+        """
+        if paths:
+            assert isinstance(paths, list), 'paths must be a list'
+            for path in paths:
+                self.toggle_emit(emit=emit, path=path)
+
+    def toggle_emit(self, emit=False, path=None):
+        """
+        Turn on/off emits for all inner nodes of path.
+        """
+        if path:
+            assert isinstance(path, tuple), 'path must be a tuple'
+            target = self.get_path(path)
+            target.toggle_emit(emit=emit)
+        elif self.inner:
+            for child in self.inner.values():
+                child.toggle_emit(emit=emit)
+        else:
+            self.emit = emit
```

Comment:
Can branch nodes have an emit setting? I don't think we want to allow that, but I'm not sure we actually forbid it

Reply:
It looks like we don't support branch node emits: https://github.com/vivarium-collective/vivarium-core/blob/7af1eb0593ff05f9fb08ce3542f29585f7866307/vivarium/core/store.py#L732-L747

Reply:
Although this might allow it... https://github.com/vivarium-collective/vivarium-core/blob/7af1eb0593ff05f9fb08ce3542f29585f7866307/vivarium/core/store.py#L652

Reply:
And here as well, only for leaf nodes: https://github.com/vivarium-collective/vivarium-core/blob/7af1eb0593ff05f9fb08ce3542f29585f7866307/vivarium/core/store.py#L652